jihollen.md

```diff
@@ -0,0 +1,26 @@
+---
+name: "@jihollen"
+project: "RFID rings"
+---
+
+# Rfid rings
+
+## Summary
+
+using rfid tech to make rings be my door card
+
+## Plan
+
+get the rings - write it using the rfid copier and program the arduino board to be able to read it then i can unlock my doors the programming of the board will be the most taxing part, but i will try my best for this to work. i just have to program the arduino to detect the card.
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
+
+| Product                   | Supplier/Link                                                                             | Cost    |
+| ------------------------- | ----------------------------------------------------------------------------------------- | ------- |
+| Ring                      | https://www.amazon.com/Riversmerge-Rewritable-Frequency-13-56MHz-Black-21mm/dp/B0B3D8Y766 | $37.88  |
+| RFID copy machine         | https://www.amazon.com/Readers-Duplicator-13-56MHz-Programmer-Frequency/dp/B0818XFX9K     | $54.39  |
```

Comment:
You should be able to find a cheap RFID read / write module for the Arduino UNO, thoughts on using one of those?

Reply:
yeah i feel like https://global.microless.com/product/arduino-rc522-rfid-card-reader-module-kit-stsx0019/?currency=usd this could be used

Reply:
but it would be much easier if i just bought it locally as it is very cheap, because to my knowledge there is no physical card

Reply:
Could you try to find a local supplier for the RC522?

Reply:
yeah there is one and it would be really easy to obtain it.

Reply:
and cheaper then the online options as it does not include shipping

Reply:
perfect! could you amend your budget?

Reply:
```suggestion
```

Reply:
yes?

Reply:
thats the thing to copy the KHZ tag

Reply:
Wouldn't you be able to copy the tag by reading it with the RFID reader and then writing using the RFID writer?

Reply:
Then i have to recode it twice sure i can do that

Reply:
Following up on the above request ^^

Reply:
Following up on the above request ^^